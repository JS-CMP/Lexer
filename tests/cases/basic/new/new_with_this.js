function Outer() {
    this.Foo = function () {
        return 123;
    };
}

console.log(new new Outer().Foo()); // NEW(NEW(Outer).Foo)
