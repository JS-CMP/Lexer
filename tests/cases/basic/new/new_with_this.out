function Outer() {
    this.Foo = function () {
        return 123;
    };
}

new new Outer().Foo(); // NEW(NEW(Outer).Foo)
